### ISSUE TITLE
Error importing SIARD2

### ISSUE THREAD

**Issue by chalkos** [keeps/db-preservation-toolkit#96]
Steps to reproduce:

1. Export MSSQL server to SIARD2
2. Export SIARD2 to something else

When exporting SIARD2 the following error occurs
```text
ERROR 2015-11-12 11:16:00,295 (Main) Unexpected exception
java.lang.NullPointerException
	at com.databasepreservation.modules.siard.in.metadata.SIARD2MetadataImportStrategy.getForeignKey(SIARD2MetadataImportStrategy.java:533)
	at com.databasepreservation.modules.siard.in.metadata.SIARD2MetadataImportStrategy.getForeignKeys(SIARD2MetadataImportStrategy.java:518)
	at com.databasepreservation.modules.siard.in.metadata.SIARD2MetadataImportStrategy.getTableStructure(SIARD2MetadataImportStrategy.java:420)
	at com.databasepreservation.modules.siard.in.metadata.SIARD2MetadataImportStrategy.getTablesStructure(SIARD2MetadataImportStrategy.java:398)
	at com.databasepreservation.modules.siard.in.metadata.SIARD2MetadataImportStrategy.getSchemaStructure(SIARD2MetadataImportStrategy.java:279)
	at com.databasepreservation.modules.siard.in.metadata.SIARD2MetadataImportStrategy.getSchemas(SIARD2MetadataImportStrategy.java:260)
	at com.databasepreservation.modules.siard.in.metadata.SIARD2MetadataImportStrategy.getDatabaseStructure(SIARD2MetadataImportStrategy.java:167)
	at com.databasepreservation.modules.siard.in.metadata.SIARD2MetadataImportStrategy.loadMetadata(SIARD2MetadataImportStrategy.java:133)
	at com.databasepreservation.modules.siard.in.input.SIARDImportDefault.getDatabase(SIARDImportDefault.java:37)
	at com.databasepreservation.Main.internal_main(Main.java:77)
	at com.databasepreservation.Main.main(Main.java:46)

```

**Comment by chalkos** [comment]
Same happens when:

1. Export MySQL (sakila) to SIARD2
2. Export SIARD2 to something else

**Comment by chalkos** [comment]
Fixed by #91
